main.ipynb

```diff
@@ -50,21 +48,15 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 314,
+   "execution_count": 87,
    "metadata": {},
-   "outputs": [
-    {
-     "output_type": "stream",
-     "name": "stderr",
-     "text": "100%|██████████| 700/700 [00:07<00:00, 96.74it/s]\n"
-    }
-   ],
+   "outputs": [],
    "source": [
     "#define the variables that determine how many photons are released\n",
-    "recordTime = .003\n",
-    "rateSingleAtom = 145000\n",
-    "rateBackground = 10000\n",
-    "lifetimeOfTrap = 1\n",
+    "recordTime = .003*0+60*10**-3  #s\n",
```

Comment:
These are in the ballpark of our experimental setup. Could these units be made to be ms and KHz? Those are more natural for us.